docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube_zh.md

```diff
@@ -354,97 +439,123 @@ rollsite-7846898d6d-j2gb9        1/1     Running   0          12m
-                    rollsite: Running                      
-                  deployments:                             
-                    client: Available                      
-                    clustermanager: Available              
-                    mysql: Available                       
-                    nodemanager-0: Available               
-                    nodemanager-1: Available               
-                    python: Available                      
-                    rollsite: Available          
+通过以上的 `kubefate cluster ls` 命令, 我们得到 `fate-9999` 的集群ID是 `29878fa9-aeee-4ae5-a5b7-fd4e9eb7c1c3`, 而 `fate-10000` 的集群ID是 `dacc0549-b9fc-463f-837a-4e7316db2537`. 我们可以通过`kubefate cluster describe`查询集群的具体访问信息，
+```
+kubefate@machine:~/demo$ kubefate cluster describe 29878fa9-aeee-4ae5-a5b7-fd4e9eb7c1c3
+UUID        	29878fa9-aeee-4ae5-a5b7-fd4e9eb7c1c3
+Name        	fate-9999
+NameSpace   	fate-9999
+ChartName   	fate
+ChartVersion	v1.9.0
+Revision    	1
+Age         	54m
+Status      	Running
+Spec        	algorithm: Basic
+            	chartName: fate
+            	chartVersion: v1.9.0
+            	computing: Spark
+            	device: CPU
+            	federation: Pulsar
+            	imagePullSecrets:
+            	- name: myregistrykey
+            	ingress:
+            	  client:
+            	    hosts:
+            	    - name: party9999.notebook.example.com
+            	  fateboard:
+            	    hosts:
+            	    - name: party9999.fateboard.example.com
+            	  pulsar:
+            	    hosts:
+            	    - name: party9999.pulsar.example.com
+            	  spark:
+            	    hosts:
+            	    - name: party9999.spark.example.com
+            	ingressClassName: nginx
+            	istio:
+            	  enabled: false
+            	modules:
+            	- python
+            	- mysql
+            	- fateboard
+            	- client
+            	- spark
+            	- hdfs
+            	- nginx
+            	- pulsar
+            	name: fate-9999
+            	namespace: fate-9999
+            	nginx:
+            	  grpcNodePort: 30098
+            	  httpNodePort: 30093
+            	  route_table:
+            	    "10000":
+            	      fateflow:
+            	      - grpc_port: 30108
+            	        host: 10.182.137.144
+            	        http_port: 30103
+            	  type: NodePort
+            	partyId: 9999
+            	persistence: false
+            	podSecurityPolicy:
+            	  enabled: false
+            	pullPolicy: null
+            	pulsar:
+            	  httpNodePort: 30094
+            	  httpsNodePort: 30099
+            	  publicLB:
+            	    enabled: false
+            	  route_table:
+            	    "9999":
+            	      host: pulsar
+            	      port: 6650
+            	      sslPort: 6651
+            	    "10000":
+            	      host: 10.182.137.144
```

Comment:
Please 192.168.xxx.xxx, because there is a security check in Webank to verify if the fake IP in document.